src/modeler/AutoRest.Swagger/Model/ServiceDefinition.cs

```diff
@@ -105,6 +105,7 @@ public ServiceDefinition()
         /// <summary>
         /// Key is the object serviceTypeName and the value is swagger definition.
         /// </summary>
+        [Rule(typeof(ArmResourcePropertiesBag))]
```

Comment:
We discourage inline definitions and there is a rule that flags such violations, so this should do for us.